fc/mef.py

```diff
@@ -626,78 +369,104 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
         saving.
     plot_filename : str, optional
         Name to use for plot files. If None, use ``str(data_beads)``.
-    full : bool, optional
+    full_output : bool, optional
         Flag specifying whether to include intermediate results in the
-        output. If `full` is True, the function returns a named tuple with
-        fields as described below. If `full` is False, the function only
-        returns the calculated transformation function.
+        output. If `full_output` is True, the function returns a named
+        tuple with fields as described below. If `full_output` is False,
+        the function only returns the calculated transformation function.
 
     Returns
     -------
-    transform_fxn : function, if ``full==False``
+    transform_fxn : function, if ``full_output==False``
         Transformation function to convert flow cytometry data from channel
         units to MEF. This function has the same basic signature as the
         general transformation function specified in ``fc.transform``.
-    namedtuple, if ``full==True``
+    namedtuple, if ``full_output==True``
         ``namedtuple``, containing the following fields in this order:
         transform_fxn : function
             Transformation function to convert flow cytometry data from
             channel units to MEF. This function has the same basic
             signature as the general transformation function specified in
             ``fc.transform``.
-        clustering_res : dict
+        clustering : dict
             Results of the clustering step, containing the following
             fields:
             labels : array
                 Labels for each element in `data_beads`.
```

Comment:
You've introduced the word "element" where we were previously using "event" (i.e. a row in an NxD numpy array of flow cytometry data). In the more general `clustering_gmm` and `selection_std` that might be appropriate as the concept of an event isn't really defined for that context, but by the time you get to `get_transform_fxn`, I think we should be back to "events".